packages/firestore/src/local/memory_query_cache.ts

```diff
@@ -137,6 +137,18 @@ export class MemoryQueryCache implements QueryCache {
     return PersistencePromise.resolve(queryData);
   }
 
+  getQueryDataForTarget(
+    transaction: PersistenceTransaction,
+    targetId: TargetId
+  ): PersistencePromise<QueryData | null> {
+    this.queries.forEach((query, queryData) => {
+      if (queryData.targetId === targetId) {
+        return PersistencePromise.resolve(queryData);
+      }
+    });
```

Comment:
I don't have strong feelings, but some of these multi-tab memory-only methods do called in unit tests. This method doesn't have a specific test at this point and thus doesn't ever get called. I went with your suggestion for now, but it may make sense to add a query cache test to cover this code directly.